packages/datadog-plugin-aws-sdk/src/base.js

```diff
@@ -243,4 +270,27 @@ function getHooks (config) {
   return { request }
 }
 
+function getHostname (store, region) {
```

Comment:
Not a blocker, but wouldn't we know this from the HTTP layer? At some point we will be actually calling out to the endpoint so we for sure have the information somewhere.

Reply:
This isn't guaranteed varying on the service called and version of the sdk. The hostname we are using could be translated differently before making an outgoing http call. Also this does not exist on the DNS/TCP layer